great_expectations/data_context/store/store.py

```diff
@@ -116,17 +124,29 @@ def deserialize(self, key, value):
     def get(self, key):
         if key == StoreBackend.STORE_BACKEND_ID_KEY:
             return self._store_backend.get(key)
-        else:
+        elif self.store_backend.__class__.__name__ == "GeCloudStoreBackend":
             self._validate_key(key)
             value = self._store_backend.get(self.key_to_tuple(key))
             if value:
-                return self.deserialize(key, value)
-            else:
-                return None
+                value = self.ge_cloud_response_json_dict_to_object_json_str(
+                    response_json_dict=value
+                )
+        else:
+            self._validate_key(key)
+            value = self._store_backend.get(self.key_to_tuple(key))
+
+        if value:
+            return self.deserialize(key, value)
+        else:
+            return None
 
     def set(self, key, value):
         if key == StoreBackend.STORE_BACKEND_ID_KEY:
             return self._store_backend.set(key, value)
+        elif self.store_backend.__class__.__name__ == "GeCloudStoreBackend":
+            # skip serialization
+            self._validate_key(key)
+            return self._store_backend.set(self.key_to_tuple(key), value)
```

Comment:
Sorry - should have added comment. Did the str comparison because there was a circular import issue, but ended up solving by importing one level deeper (from `great_expectations.data_context.store.ge_cloud_store_backend` instead of `great_expectations.data_context.store`). Regarding serializer, nixed the above elif and moved to the serialize method, to determine when to output json str vs. yaml str.